dvclive/live.py

```diff
@@ -102,7 +105,7 @@ def summary_path(self):
 
     @property
     def html_path(self):
-        return str(self.dir) + "_dvc_plots/index.html"
+        return os.path.join(self.dir, "report.html")
```

Comment:
This is what closes #195 . Report is now generated in `dvclive/report.html`.

Reply:
Will it be an issue if the dvclive dir is marked as a plots output and there is already an html file inside it?

Reply:
> Will it be an issue if the dvclive dir is marked as a plots output and there is already an html file inside it?

Not sure if an issue but the HTML will be tracked by whichever config they set for the `plots` output (so HTML might be added to DVC cache).

It's not strictly related with this P.R. but one of the directions I also wanted with deprecating `live` was to encourage more granular output config for DVCLive outputs which is not currently possible:

```yaml
...
plots:
  - dvclive/scalars
      cache: false
  - dvclive/images
metrics:
  # I mean to open issue to discuss also nesting summary
  - dvclive/summary.json:
      cache: false
```

```
#.gitignore
dvclive/report.html
```

Reply:
Also, looks like the dvc output will be wrong once this is merged:

```
Live summary will be created at:
/Users/dave/Code/dvc-checkpoints-mnist/dvclive_dvc_plots/index.html
```

How do we sequence these PRs to keep everything working as expected? Would it be better to close #295 in a follow-up PR?

Reply:
A subsequent DVC P.R. should be created to fix that message but more importantly to deactivate the HTML generation on DVC side. 

`dvclive` is pinned in DVC so I see the sequence as:

Merging https://github.com/iterative/dvc/pull/7409 > Approval of this P.R. > New P.R. in DVC fixing message and disabling generation.

Reply:
One other discrepancy I noticed:

https://github.com/iterative/dvc/blob/6a6955e27cf69490c7f694980dadfd3f7a5f9949/dvc/repo/live.py#L22

The dvc-generated html includes metrics, but this PR does not. Is there a plan to include metrics in a future PR?

Reply:
> deactivate the HTML generation on DVC side

Should html still be generated now in `dvclive_dvc_plots`? I'm not seeing that happen.

Reply:
I'm confused what's happening now in `dvclive_dvc_plots` with these changes. It doesn't look like it's being updated live anymore after this PR.

Reply:
> I'm confused what's happening now in dvclive_dvc_plots with these changes. It doesn't look like it's being updated live anymore after this PR.

If you install this P.R. and https://github.com/iterative/dvc/pull/7514 (follow up for DVC once this P.R. gets merged), then :

- `dvclive_dvc_plots` should not be generated at all
- The new HTML report (with correct refreshing) should be generated at `{dvclive_dir}/report.html`.

Reply:
If we are separating this, we might as well remove `dvc/commands/live` functionality. We created for visualizing `dvclive` results, but  now `dvclive` will be able to do it by itself.